specification.markdown

```diff
@@ -404,23 +404,36 @@ PathMaxOccurrence     = 1*DIGIT ; no upper limit
+The GlobalParentOccurence part is interpreted as an ABNF Variable Repetition.
+The repetition amounts correspond to the amount of unspecified valid Elements that can be found in place of the GlobalPlaceholder for the path to be valid.
 
-PathMaxOccurrence represents the maximum number of element path possible between the EBMLFixedParent and the Global Element EBMLElementPath. It cannot have the value 0 as it would be the Global Element can only be found right after the EBMLFixedParent, in which case it's not a Global Element anymore. If PathMaxOccurrence is not present then there is no upper bound for the permitted number of occurrences of element path possible between the EBMLFixedParent and the Global Element EBMLElementPath.
+PathMinOccurrence represents the minimum number of path elements required between the EBMLParentPath and the EBMLLastPath.
+If PathMinOccurrence is not present then that GlobalPlaceholder has a PathMinOccurrence value of 0.
 
-If the path contains an EBMLPathAtomRecursive part, the EBML Element can occur within itself recursively (see (#recursive)).
+For example consider an EBML Path `\a(1*\)\global` there has to be at least one path element between the EBMLParentPath `\a` and the EBMLLastPath `\global`.
+So the `global` EBML Element cannot be found inside the `\a` EBML Element as it means the resulting path `\a\global` has zero path element between the EBMLParentPath `\a` and the EBMLLastPath `\global`.
+But the `global` EBML Element can be found inside the `\a\b` EBML Element, or inside the `\a\b\c` EBML Element, or inside the `\a\b\c\d` EBML Element, etc.
 
-As an example, a `path` of `1*(\Segment\Info)` means the element Info is found inside the Segment elements at least once and with no maximum iteration. An element SeekHead with path `0*2(\Segment\SeekHead)` may not be found at all in its Segment parent, once or twice but no more than that.
+PathMaxOccurrence represents the maximum number of path elements possible between the EBMLParentPath and the EBMLLastPath.
+If PathMaxOccurrence is not present then there is no upper bound for the permitted number of occurrences of path elements possible between the EBMLParentPath and the EBMLLastPath.
+PathMaxOccurrence cannot have the value 0 as it would mean the EBMLLastPath can only be found right after the EBMLParentPath, in which case it's not a Global Element anymore and a GlobalPlaceholder MUST NOT be used. 
+
+For example consider an EBML Path `\a(0*1\)\global`, there has to be at most one path element between the EBMLParentPath `\a` and the EBMLLastPath `\global`.
+So the `global` EBML Element can be found inside the `\a` EBML Element or inside the `\a\b` EBML Element.
```

Comment:
Like the comment above, I'm not sure how to understand this if it was `(0*1\)\global`.